src/utils/dev.js

```diff
@@ -11,3 +11,14 @@ export const uniqueId = (() => {
     return map.get(object);
   };
 })();
+
+export const isMobile = () => {
+  if (sessionStorage.desktop) return false;
```

Comment:
these properties are new to me -- if you're getting them from some online documentation that explains how they're set, and across which browsers, please add a link to that documentation in a comment here

Reply:
i created `utils/dev` for functions that would only be used in development--if something is going to be part of the production code, it should probably go elsewhere. perhaps in a new `utils/browser` (IF it still needs to be used--see below)

Reply:
That function was put together as a blend of [this](https://stackoverflow.com/a/16080250) and [this](https://stackoverflow.com/a/49425890). 

I have removed this in favour of the approach below